src/cmd/vm.rs

```diff
@@ -0,0 +1,20 @@
+// Copyright 2023 The ChromiumOS Authors
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+use anyhow::Result;
+use argh::FromArgs;
+
+#[derive(FromArgs, PartialEq, Debug)]
+/// create a virtual machine
```

Comment:
Move doc comment above the derive macros

Reply:
Thank you for your review!  
In the [official documentation](https://docs.rs/argh/latest/argh/) of the argh crate and other lium commands, the doc comment is written between the derive macro and the argh macro, but is it okay to not follow these?

Reply:
+1 for @4n12i . I think keep the consistency with the other usage is a good idea.

Reply:
Apologies I didn't see that!  Keep it consistent!